.github/workflows/build-docker-image.yml

```diff
@@ -23,9 +23,13 @@ jobs:
     # Setup for buildx
     - name: Set up QEMU
       uses: docker/setup-qemu-action@v1
+      with: 
+        image: tonistiigi/binfmt:latest
```

Comment:
Is there a tagged version of it? Or amy other way to pin it?

Reply:
Still WIP! 😆